src/accessors.d

```diff
@@ -41,7 +41,8 @@ mixin template GenerateFieldAccessorMethods()
 
         foreach (name; Filter!(isNotThis, __traits(derivedMembers, typeof(this))))
         {
-            alias field = Alias!(__traits(getMember, typeof(this), name));
+            enum field_str = "Alias!(__traits(getMember, typeof(this), \""~name~"\"))";
```

Comment:
Follow "The D Style": https://dlang.org/dstyle.html

- camelCase `enum` names: `fieldString`
- You can use other quotes to avoid escaping: \`Alias!(__traits(getMember, typeof(this), "\` ~ name ~ \`"))\`
- Space between binary operators (` ~ name ~ `)
- IMHO it is better to have the explicit type for the `enum` (`string`)